pageTests/trust-admin/add-a-ward-success.test.js

```diff
@@ -1,12 +1,12 @@
-import { getServerSideProps } from "../../pages/trust-admin/add-a-ward-success";
+import { getServerSideProps } from "../../pages/trust-admin/wards/[id]/add-success";
```

Comment:
Do we need to rename this file too now that the original page has moved? (and the other page tests in this PR)

Reply:
Yeah, so I forgot to move the corresponding test files, including the hospitals ones, to follow the structure. I'm doing that now and will submit a new PR with the changes.